pages/modules/[module].tsx

```diff
@@ -73,43 +77,58 @@ const ModulePage: NextPage<ModulePageProps> = ({
                 <div>
                   <ul className="mt-4">
                     {versionInfos.reverse().map((version) => (
-                      <li
-                        key={version.version}
-                        className="border rounded p-2 mt-2 flex items-stretch gap-4"
-                      >
-                        <Link href={`/modules/${module}/${version.version}`}>
-                          <a>
-                            <div className="rounded-full border h-14 w-14 grid place-items-center hover:border-gray-800">
-                              {version.version}
-                            </div>
-                          </a>
-                        </Link>
-                        <div className="flex flex-1 justify-between">
-                          <div className="flex flex-col justify-between">
-                            <div />
-                            <div className="self-end text-gray-500">
-                              compatibility level{' '}
-                              {version.moduleInfo.compatibilityLevel}
+                      <>
+                        <li
+                          key={version.version}
+                          className="border rounded mt-2"
+                        >
+                          {version.isYanked && (
+                            <div
+                              key={`${version.version}-yanked`}
+                              className="p-2 mb-2 bg-amber-300"
+                            >
+                              Version yanked with comment:{' '}
```

Comment:
can we link from `yanked` to somewhere that explains the concept, so end users can know what this means?

Reply:
I did look around a bit, but AFAIK there isn't really a page that explains the concept ATM.

I'll open up an issue here and in Bazel itself to do this as a follow-up.

Reply:
Issue for follow-up: https://github.com/bazel-contrib/bcr-ui/issues/30